src/test/java/com/github/neatlife/jframework/util/PrecisionUtilTest.java

```diff
@@ -0,0 +1,137 @@
+package com.github.neatlife.jframework.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.math.BigDecimal;
+
+public class PrecisionUtilTest {
+
+    @Test
+    public void testBigDecimalAdd() {
+        Assert.assertEquals(BigDecimal.ONE,
+                PrecisionUtil.add(null, BigDecimal.ONE));
```

Comment:
Add test where neither param is null, and in both tests with nulls change the non-null param to something other than `ONE` so it's clear that the non-null param is not the source of the return value.

Reply:
Done